src/mobile/apps/show/templates/index.jade

```diff
@@ -38,3 +38,5 @@ block content
     if show.get('press_release')
       include press_release
 
+  - if (jsonLD)
```

Comment:
@sepans Can't tell if this is weird github formatting or not, but this `if` looks suspect with the `-`. And the `include` line may need to be indented.

Reply:
I took it from the `desktop` file which also was my code. Played with it a bit and realized when adding`()` around `jsonLD`, `-` is required otherwise not needed (have no idea why 😄 )